src/components/layout/MainSidebar/SidebarNavItem.js

```diff
@@ -2,32 +2,36 @@ import React from "react";
 import PropTypes from "prop-types";
 import { NavLink as RouteNavLink } from "react-router-dom";
 import { NavItem, NavLink } from "shards-react";
+import { useTranslation } from "react-i18next";
 
-const SidebarNavItem = ({ item }) => (
-  <NavItem>
-    <NavLink tag={RouteNavLink} to={item.to}>
-      {item.htmlBefore && (
-        <div
-          className="d-inline-block item-icon-wrapper"
-          dangerouslySetInnerHTML={{ __html: item.htmlBefore }}
-        />
-      )}
-      {item.title && <span>{item.title}</span>}
-      {item.htmlAfter && (
-        <div
-          className="d-inline-block item-icon-wrapper"
-          dangerouslySetInnerHTML={{ __html: item.htmlAfter }}
-        />
-      )}
-    </NavLink>
-  </NavItem>
-);
+const SidebarNavItem = ({ item }) => {
+  const { t } = useTranslation();
+  return (
+    <NavItem>
+      <NavLink tag={RouteNavLink} to={item.to}>
+        {item.htmlBefore && (
+          <div
+            className="d-inline-block item-icon-wrapper"
+            dangerouslySetInnerHTML={{ __html: item.htmlBefore }}
+          />
+        )}
+        {t(item.title) && <span>{t(item.title)}</span>}
+        {item.htmlAfter && (
+          <div
+            className="d-inline-block item-icon-wrapper"
+            dangerouslySetInnerHTML={{ __html: item.htmlAfter }}
+          />
+        )}
+      </NavLink>
+    </NavItem>
+  );
+};
 
 SidebarNavItem.propTypes = {
   /**
    * The item object.
    */
-  item: PropTypes.object
+  item: PropTypes.object,
```

Comment:
Hi @amohdsharfuddin , is there specific reason to add the comma here or it was left by accident? ;)

Reply:
That is just due to auto format on save extension. I changed some lines but I undo the changes because it was not needed so when I save the file it auto formats.

Reply:
I see. If it is caused by an auto-formatting tool then let it make the changes because it ensures consistency which is what we want to achieve. Make sure that everyone is using the same formating tool and formats though, we want to have the same consistency in the project. Feel free to document it in the Confluence on which tools and what format (or default format) you used for the project. ;)

Reply:
Hmm, take a look at this documentation from NPM. https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md
I think it worth to check in to repository.